include/QtNodes/internal/GraphicsView.hpp

```diff
@@ -41,10 +41,13 @@ class NODE_EDITOR_PUBLIC GraphicsView
 
   /// @brief max=0/min=0 indicates infinite zoom in/out
   void
-  setScaleRange(double minimum = 0, double maximum = 0);
+  setScaleRange(const double minimum = 0, const double maximum = 0);
```

Comment:
const in POD members are considered pedantic.

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f16-for-in-parameters-pass-cheaply-copied-types-by-value-and-others-by-reference-to-const

Reply:
Moreover, it lack consistency of the codebase. Or all the POD data are not passed by const or none of that should be.